Flow.hs

```diff
@@ -136,8 +144,17 @@ f .> g = compose f g
     -0.25
 
     Or use it anywhere you would use ('Prelude..').
+
+    Note that this operator's precedence is lower than ('.>'), so they can be
+    used together.
+
+    -- ((1 + 2) * 3) ^ 4
+    >>> ((^ 4) <. (+ 2) .> (* 3)) 1
+    6561
+    >>> ((* 3) .> (^ 4) <. (+ 2)) 1
+    6561
 -}
-infixr 9 <.
+infixr 8 <.
```

Comment:
Changing the precedence of this operator makes it not a drop-in replacement for (`.`). [A few operators](http://hackage.haskell.org/package/haskell-src-exts-1.16.0.1/docs/src/Language-Haskell-Exts-Fixity.html#preludeFixities) have precedence 8: `^`, `^^`, and `**`. Considering that these composed functions are hard to understand in the first place, I think I should leave `<.` at precedence 9.